cookbooks/aws-parallelcluster-slurm/recipes/install/install_pyxis.rb

```diff
@@ -35,12 +38,59 @@
     set -e
     tar xf #{pyxis_tarball} -C /tmp
     cd /tmp/pyxis-#{pyxis_version}
-    CPPFLAGS='-I /opt/slurm/include/' make
-    CPPFLAGS='-I /opt/slurm/include/' make install
-    mkdir -p /opt/slurm/etc/plugstack.conf.d
-    echo -e 'include /opt/slurm/etc/plugstack.conf.d/*' | tee /opt/slurm/etc/plugstack.conf
-    ln -fs /usr/local/share/pyxis/pyxis.conf /opt/slurm/etc/plugstack.conf.d/pyxis.conf
+    CPPFLAGS='-I #{node['cluster']['slurm']['install_dir']}/include/' make
+    CPPFLAGS='-I #{node['cluster']['slurm']['install_dir']}/include/' make install
   PYXIS_INSTALL
   retries 3
   retry_delay 5
 end
+
+# Spank configurations
+
+directory node['cluster']['slurm']['spank_config_dir'] do
```

Comment:
Shall we avoid creating this directory if we are not configuring Slurm?

Reply:
We can, but if we do, the user will need to do it as part of Pyxis enablement. 
It think we may want to minimize the effort on the end user, right?
Do you see drawbacks in creating it?